usagov/usagov-2021
# Usagov 1791 analytics attribute

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1791

## Description

Add a data attribute to links in the benefit search results so that analytics can track clicks without relying on other markup.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Need to have working benefit search result pages.
Import the new views.

### Change Requirements



### Validation Steps

- On the English benefit search page, select all categories.
- Confirm life event links have markup with the data-attribute set
```
<a href="/benefit-finder/disability" data-analytics="category-results-links" hreflang="en">Benefit finder: disability</a>
```
- Confirm page links have markup with the same data-attribute
- Repeat on Spanish pages, only markup difference should be `hreflang="es"`

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
